app/resources/translations/cs_CZ/messages.cs_CZ.yml

```diff
@@ -7,75 +7,411 @@
 #          process and your PR so that it will eventually be necessary to
 #          re-map your translations.
 
-#   0 Unused messages
-#   8 Legacy untranslated messages
-#  18 Untranslated messages
-#  32 Legacy translation messages
-# 540 Translation messages
+# 361 Unused messages
+#   5 Legacy untranslated messages
+#   8 Untranslated messages
+#  20 Legacy translation messages
+# 197 Translation messages
+
+#--- Unused messages ----------------------------------------------------------
+
+"(default template)": "(standardní šablona)"
+"(no category)": "(bez kategorie)"
+"(no group)": "(nezařazení do skupiny)"
+"(none)": "(nic)"
+"Add some sample <a href='%url%' class='btn btn-default'>Records with Loripsum text</a>": "Přidat ukázku <a href='%url%' class='btn btn-default'>Záznam s Lorem ipsum</a>"
+"Current (saved) status:": "Současný (uložený) stav:"
+"Do you really want to delete %FOLDERNAME%?": "Opravdu smazat %FOLDERNAME%?"
+"Excerpt": "Souhrn"
+"Expand sidebar": "Roztáhnout postranní lištu"
+"The settings for roles and permissions are stored in <code>%permissions%</code>.": "Nastavení uživatelský práv a rolí je v <code>%permissions%</code>."
+"Title": "Nadpis"
+"View (saved version) on site": "Zobrazit uloženou verzi na webu"
+"contenttypes.generic.actions-all": "Akce pro %contenttypes%"
```

Comment:
Key names like this one `contenttypes.generic.actions-all` should not be quoted, as you'll see below the removed line is:

```diff
-contenttypes.generic.actions-all: "Akce pro %contenttypes%"
```

Reply:
This is an action done by Bolt and it will do it for any edited/saved language file.
So this shouldn't be a block issue for this PR.
What will be the proper action here?

Reply:
I'm here to help. I'm trying to understand this.
I will do a test with a plain install and will report back regarding those strings